src/Violation/Renderer/Html/HtmlRendererFactory.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace SensioLabs\DeprecationDetector\Violation\Renderer\Html;
+
+
+use SensioLabs\DeprecationDetector\Violation\Renderer\MessageHelper\MessageHelper;
+use Symfony\Component\Filesystem\Filesystem;
+
+class HtmlRendererFactory
```

Comment:
the same as [this comment](https://github.com/sensiolabs-de/deprecation-detector/pull/58/files#r43491096)